device/components/RTA_helpers.brs

```diff
@@ -321,75 +321,91 @@ end function
 		else
 			RTA_logWarn("tried to call getParent() on non node of type " + type(callOn))
 		end if
-	else if functionName = "count()" then
+	else if functionName = "count" then
 		if RTA_isArray(callOn) OR RTA_isKeyedValueType(callOn) then
 			return callOn.count()
 		else
 			RTA_logWarn("tried to call count() on non AA or array of type " + type(callOn))
 		end if
-	else if functionName = "keys()" then
+	else if functionName = "keys" then
 		if RTA_isNode(callOn) then
 			return callOn.keys().toArray() ' keys() returns an roList when called on a node. We have to convert to array as all of our array checks are specifically looking for an array
 		else if RTA_isAA(callOn) then
 			return callOn.keys()
 		else
 			RTA_logWarn("tried to call keys() on non keyed value of type " + type(callOn))
 		end if
-	else if functionName = "len()" then
+	else if functionName = "len" then
 		if RTA_isString(callOn) then
 			return callOn.len()
 		else
 			RTA_logWarn("tried to call len() on non string of type " + type(callOn))
 		end if
-	else if functionName = "getChildCount()" then
+	else if functionName = "getChildCount" then
 		if RTA_isNode(callOn) then
 			return callOn.getChildCount()
 		else
 			RTA_logWarn("tried to call getChildCount() on non node of type " + type(callOn))
 		end if
-	else if functionName = "threadinfo()" then
+	else if functionName = "threadinfo" then
 		if RTA_isNode(callOn) then
 			return callOn.threadinfo()
 		else
 			RTA_logWarn("tried to call threadinfo() on non node of type " + type(callOn))
 		end if
-	else if functionName = "getFieldTypes()" then
+	else if functionName = "getFieldTypes" then
 		if RTA_isNode(callOn) then
 			return callOn.getFieldTypes()
 		else
 			RTA_logWarn("tried to call getFieldTypes() on non node of type " + type(callOn))
 		end if
-	else if functionName = "subtype()" then
+	else if functionName = "subtype" then
 		if RTA_isNode(callOn) then
 			return callOn.subtype()
 		else
 			RTA_logWarn("tried to call subtype() on non node of type " + type(callOn))
 		end if
-	else if functionName = "boundingRect()" then
+	else if functionName = "boundingRect" then
 		if RTA_isNode(callOn) then
 			return callOn.boundingRect()
 		else
 			RTA_logWarn("tried to call boundingRect() on non node of type " + type(callOn))
 		end if
-	else if functionName = "localBoundingRect()" then
+	else if functionName = "localBoundingRect" then
 		if RTA_isNode(callOn) then
 			return callOn.localBoundingRect()
 		else
 			RTA_logWarn("tried to call localBoundingRect() on non node of type " + type(callOn))
 		end if
-	else if functionName = "sceneBoundingRect()" then
+	else if functionName = "sceneBoundingRect" then
 		if RTA_isNode(callOn) then
 			return callOn.sceneBoundingRect()
 		else
-			RTA_logWarn("tried to call sceneBoundingRect() on non node of type " + type(callOn))
+			RTA_logWarn("tried to call sceneBoundingRect on non node of type " + type(callOn))
+		end if
+	else if functionName = "sceneSubBoundingRect" then
```

Comment:
Is there a way we could add better support for _any_ function, rather than needing to hardcode specific functions? This list might become a bit unmanageable over time. Just curious.

Reply:
I believe that was one of the things Roku was asking us to rank. Currently there is no way to call a function from a param. I wouldn't be surprised if we never get it though as they use the fact that you have to use an exact string for a function call in their static analysis checks.